db/migrate/20191015133100_update_institution_programs_indexes.rb

```diff
@@ -0,0 +1,17 @@
+class UpdateInstitutionProgramsIndexes < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    remove_index :institution_programs, name: "index_institution_programs"
+
+    add_index :institution_programs,
+              [:institution_id],
+              using: :btree,
+              algorithm: :concurrently
+
+    add_index :institution_programs,
+              [:description, :version],
```

Comment:
We're removing `version`, so we don't need to index on it.

Reply:
I wasn't planning on getting rid of `version`.  Maybe I'm missing something, but it's still a search parameter used by vets-website, so I think we'd still want to use it in the index.  Also I think we'd still want it for archiving.

Reply:
We're removing it for the same reason we're removing `facility_code`:  it's on the `institutions` record and will be searchable there.